flair/datasets/treebanks.py

```diff
@@ -1722,3 +1722,30 @@ def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True, s
         )
 
         super(UD_LATVIAN, self).__init__(data_folder, in_memory=in_memory, split_multiwords=split_multiwords)
+        
+class UD_Lithuanian(UniversalDependenciesCorpus):
```

Comment:
Can you capitalize the object name like the other datasets, i.e. "UD_LITHUANIAN" instead of "UD_Lithuanian"

Reply:
Sure, i did it. Could you check one more time?